ovh/error.go

```diff
@@ -4,8 +4,12 @@ import "fmt"
 
 // APIError represents an error that can occurred while calling the API.
 type APIError struct {
```

Comment:
Don't we need to put json tags ?

In my understanding, `json.Unmarshal` will look for keys `Message`, `Class`, `Details`, and `Code`, where in OVH API, we are returning `message`, `class`, and `details`

This code https://github.com/ovh/go-ovh/blob/302dfc012714871f55e4ce3bae69f903fe28327b/ovh/ovh.go#L407 will then unmarshal, but probably returns an empty message.

What do you think ?

Reply:
When I tested it, it seemed to me that it ignored the case of the keys. But I can definitely add the json tags to make it clearer.